hono/.dev.vars.example

```diff
@@ -0,0 +1 @@
+DATABASE_URL=
```

Comment:
Is it intentional for this file to be a one-liner?

Reply:
Yes, see README. This is a template file that should be updated by the developer.